backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java

```diff
@@ -159,6 +159,55 @@ public ApiFacility addFacilityNew(
     return new ApiFacility(created);
   }
 
+  public ApiFacility addAndReturnFacility(
```

Comment:
also this might be the time to incorporate an input object for adding/updating facility

Reply:
The difference between `addFacilityNew` and `addAndReturnFacility` is that `addAndReturnFacility` returns a `Facility` object instead of a `String`. We can't use the `addFacilityNew` because we need the `id` of the `Facility` object to update the various pieces in the UI.

Reply:
The UI is not using `addFacilityNew.` Functionally, here in the resolver, `addAndReturnFacility` is the same as `addFacilityNew`, but they return different things in the schema.

Reply:
since the UI on main/prod is using addFacilityNew, can we reuse the addFacility endpoint for this, instead of creating a new endpoint? you can change the addFacility endpoint to be exactly tje same as addAndReturnFacility.

Reply:
the ideal state is the have the UI use an endpoint named `addFacility` (new endpoints such as `addFacilityNew` are only temporary to maintain backward compatibility) so currently the UI in prod/main using `addFacilityNew`, the normal flow of operations is to 

1. push a PR to make `addFacility` exactly mirror `addFacilityNew` and switch the ui to use `addFacility`
2. push a PR to remove `addFacilityNew` after a few days or so that way we dont have any old clients trying to access the old query

Since you are creating a new query `addAndReturnFacility` and we are not using `addFacility` atm, we can

1. this PR can make `addFacility` return the facility, change the UI to use  `addFacility`
2. few days after, a second PR to remove `addFacilityNew`

Let me know if you have any questions